Pat.Subscriber/BatchProcessing/ActionInvocationBatchProcessingBehaviour.cs

```diff
@@ -6,12 +6,12 @@
 
 namespace Pat.Subscriber.BatchProcessing
 {
-    public class DefaultBatchProcessingBehaviour : IBatchProcessingBehaviour
+    public class ActionInvocationBatchProcessingBehaviour : IBatchProcessingBehaviour
```

Comment:
This rename is a breaking change, can we avoid that somehow?

Perhaps `DefaultBatchProcessingBehaviour` could inherit from `ActionInvocationBatchProcessingBehaviour`. Mark `DefaultBatchProcessingBehaviour` as obsolete and look to remove in a later revision? The code in `EnsureTerminatingBehavioursAdded` would need to be updated accordingly.